lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -488,6 +488,15 @@ def get_edgecolor(self):
         return self._maybe_depth_shade_and_sort_colors(super().get_edgecolor())
 
 
+def get_data_scale(X, Y, Z):
```

Comment:
Please rename this to `_get_data_scale` to make it clear that it is a private method. A short doc-string would also be useful, possibly single-line.